### PR TITLE
feat(movie/ratings): added imdb (Radarr Proxy)

### PR DESCRIPTION
**Pull request by marcofaggian**
#### Description
Hi, taking inspiration from the [Radarr approach](https://github.com/Radarr/Radarr/blob/5ec7e86488755f759beb205d6cb2755867e05269/src/NzbDrone.Common/Cloud/RadarrCloudRequestBuilder.cs#LL23C16-L23C16) for [implementing IMDB Ratings](https://github.com/Radarr/Radarr/blob/5ec7e86488755f759beb205d6cb2755867e05269/src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs#L159), I've added support for IMDB ratings.

To enable this, I've explicited the `RatingsResponse` type, decoupling it from the RottenTomatoes response.
In addition to that, I moved the RT and IMDB APIs to a ratings specific folder.

This changes should enable an easier maintenance and adoption of multiple ratings APIs.

#### Screenshot (if UI-related)
immagine

#### To-Dos

- [x] Successful build `yarn build`
- [ ] Translation keys `yarn i18n:extract`
- [ ] Database migration (if required)

#### Issues Fixed or Closed

- Fixes #XXXX

### CODE REVIEW

**Comment by Fallenbagel** [comment]
Thanks so much for the PR! Would take a look first thing in the morning c:

**Review by Fallenbagel — APPROVED** [review]

File: server/api/rating/rottentomatoes.ts [hunk 1]
```diff
@@ -140,13 +140,17 @@ class RottenTomatoes extends ExternalAPI {
 
       return {
         title: movie.title,
+
```
Comment (fallenbagel) [inline server/api/rating/rottentomatoes.ts:140]: Prettier needs to be run to pass the format test and my guess is this extra space is causing the issue

**Review by Fallenbagel — APPROVED** [review]

**Comment by marcofaggian** [comment]
@Fallenbagel Passed tests 🎉 now is should include #401, thanks @darmiel for that!

**Comment by Fallenbagel** [comment]
@marcofaggian could you resolve the merge conflict?

**Comment by marcofaggian** [comment]
@Fallenbagel good to go?

**Comment by Fallenbagel** [comment]
> @Fallenbagel good to go?

Just a question on .vscode stuff. Is that necessary for this PR?

**Comment by marcofaggian** [comment]
Sorry, an intruder that i missed

**Comment by Fallenbagel** [comment]
@marcofaggian hi. Could you also fix/remove the RT comments from the imdbradarrproxy file?

Also as [overseerr](https://github.com/sct/overseerr) is also interested in this, let's redirect this PR there and we can pull it in.

**Comment by marcofaggian** [comment]
Ok, it seems a good plan.

**Comment by bin101** [comment]
Time to merge this?

**Comment by Fallenbagel** [comment]
> Time to merge this?

We will be pulling in the commits from upstream.as there were a few changes made so that supersedes this PR

**Comment by Uruknara** [comment]
@Fallenbagel you can close this one

**Comment by Fallenbagel** [comment]
This was superseded by [this](https://github.com/Fallenbagel/jellyseerr/commit/b4191f9c65b7ff08764e61d18e7a75bc8d4b3325)

**Pull request closed without merging**